calicoctl/commands/config.go

```diff
@@ -0,0 +1,319 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"errors"
+	"os"
+	"strings"
+
+	"github.com/docopt/docopt-go"
+	"github.com/projectcalico/libcalico-go/lib/client"
+	"github.com/projectcalico/calico-containers/calicoctl/commands/constants"
+	"github.com/projectcalico/calico-containers/calicoctl/commands/clientmgr"
+	"fmt"
+	"strconv"
+)
+
+func Config(args []string) {
+	doc := constants.DatastoreIntro + `Usage:
+  calicoctl config set <NAME> <VALUE> [--node=<NODE>] [--config=<CONFIG>]
+  calicoctl config unset <NAME> [--node=<NODE>] [--config=<CONFIG>]
+  calicoctl config view <NAME> [--node=<NODE>] [--config=<CONFIG>]
```

Comment:
Feels inconsistent with "show" for other commands, can you check that we're matching k8s here or bring it into line with our other commands?